.github/workflows/element-build-and-test.yaml

```diff
@@ -88,6 +88,19 @@ jobs:
             cypress/videos
             cypress/synapselogs
 
+      - name: Store benchmark result
+        uses: matrix-org/github-action-benchmark@jsperfentry-1
+        with:
+          name: Cypress measurements
+          tool: 'jsperformanceentry'
+          output-file-path: cypress/performance/measurements.json
+          # The dashboard is available at https://matrix-org.github.io/matrix-react-sdk/cypress/bench/
+          benchmark-data-dir-path: cypress/bench
+          fail-on-alert: false
+          comment-on-alert: false
+          github-token: ${{ secrets.DEPLOY_GH_PAGES }}
+          auto-push: ${{ github.ref == 'refs/heads/develop' }}
```

Comment:
Ah you're right, as currently configured, this step only actually does something when on `develop`. Perhaps in the future, the team may want to enable the comment and / or fail on regression modes, so that any regressions are made obvious on PRs before they land.

Assuming it's best to start more conservatively and just build up data on develop for now, I'll change this to add the `if` like you suggest.

Reply:
Great, that's how I understood it too